frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -829,11 +829,11 @@ export function ContainerInfo(props: ContainerInfoProps) {
 export interface OwnedPodsSectionProps {
   resource: KubeObjectInterface;
   hideColumns?: PodListProps['hideColumns'];
-  noSearch?: boolean;
```

Comment:
We need to mark such changes as deprecated (and add a warning when they are used) rather than removing them. Otherwise we can break things, even if it's at a typing check level.

Reply:
kept the name